modules/ocf_kubernetes/manifests/secrets.pp

```diff
@@ -0,0 +1,45 @@
+class ocf_kubernetes::secrets {
```

Comment:
please create a worker directory and put this file there: `ocf_kubernetes::worker::secrets` 

Did this for the master with ingress stuff